netweaver/defaults.yaml

```diff
@@ -1,7 +1,10 @@
 netweaver:
+  clean_nfs: True
+  installation_folder: /tmp/unattended
```

Comment:
@arbulu89 Maybe this name is too generic that could collide with an existing folder. What if `/tmp/swpm_uanttended`

Reply:
`/tmp/netweaver_unattended`?